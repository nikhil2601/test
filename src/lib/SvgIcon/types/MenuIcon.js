import React from 'react';
import PropTypes from 'prop-types';

import SvgIcon from '../SvgIcon';

const MenuIcon = ({ setRef, ...rest }) => (
    <SvgIcon {...rest} ref={setRef}>
        <path d="M3,6H21V8H3V6M3,11H21V13H3V11M3,16H21V18H3V16Z" />
    </SvgIcon>
);

MenuIcon.propTypes = {
    setRef: PropTypes.func,
    viewBox: PropTypes.string,
};

MenuIcon.defaultProps = {
    setRef: null,
    viewBox: '0 0 24 24',
};

export default MenuIcon;
