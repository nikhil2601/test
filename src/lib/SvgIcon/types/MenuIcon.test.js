import React from 'react';
import { shallow } from 'enzyme';

import MenuIcon from './MenuIcon';

describe('<MenuIcon />', () => {
    const wrapper = shallow(<MenuIcon />);

    it('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
