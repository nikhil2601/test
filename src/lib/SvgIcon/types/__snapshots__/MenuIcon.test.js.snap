// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuIcon /> should render correctly 1`] = `
<SvgIcon
  color="inherit"
  height="1rem"
  margin={null}
  opacity={null}
  padding={null}
  titleAccess={null}
  viewBox="0 0 24 24"
  width="1rem"
>
  <path
    d="M3,6H21V8H3V6M3,11H21V13H3V11M3,16H21V18H3V16Z"
  />
</SvgIcon>
`;
